.github/workflows/gcp_trustedboot_tpm2_tests.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#set -Eeuo pipefail
+
+# Name of Image to test
+cname="${@: -1}"
+
+configFile="gcp_test_config.yaml"
+containerName="ghcr.io/gardenlinux/gardenlinux/integration-test:today"
+artifact_dir="/tmp/gardenlinux-build-artifacts"
+platform_test_log_dir="/tmp/gardenlinux-platform-test-logs"
+
+mkdir -p "$platform_test_log_dir"
```

Comment:
Please update according to latest changes in `gcp_tets.sh`.
See also `gcp_trustedboot_tests.sh`.